docs/advanced/local_extensions.rst

```diff
@@ -54,6 +54,28 @@ It's likely that we'd only want to register a single function as a filter. For t
 
 This snippet will achieve the exact same result as the previous one.
 
+Similarly, to register a single function as a test extension we can use
+the ``simple_test`` decorator:
+
+.. code-block:: json
+
+    {
+        "project_slug": "Foobar",
+        "year": "{% now 'utc', '%Y' %}",
+        "_extensions": ["local_extensions.simpletestextesion"]
+    }
+
+.. code-block:: python
+
+    from cookiecutter.utils import simple_test
+
+
+    @simple_test
+    def simpletestextesion(v):
+        return v in [True, "True", "true]
```

Comment:
```suggestion
        return v in [True, "True", "true"]
```

Reply:
*sigh*  I really need to proofread my own stuff :-(